client/my-sites/sidebar/tools-menu.jsx

```diff
@@ -49,7 +49,6 @@ class ToolsMenu extends PureComponent {
 			label: translate( 'Plugins' ),
 			capability: 'manage_options',
 			queryable: ! config.isEnabled( 'calypsoify/plugins' ) || ! isAtomicSite,
-			config: 'manage/plugins',
```

Comment:
We should be able to remove the config logic on https://github.com/Automattic/wp-calypso/pull/53455/files#diff-0e72ae8e54cb18e6af045f21c5f2f05101cb619e76bc2a39cbcf4ca6f5e4143fR110 right?